File: app/src/main/java/com/aero51/moviedatabase/repository/model/credits/Actor.java
package com.aero51.moviedatabase.repository.model.credits;

import androidx.room.Entity;
import androidx.room.PrimaryKey;
@Entity(tableName = "actor")
public class Actor {

    private String birthday;
    @PrimaryKey(autoGenerate = false)
    private Integer id;
    private String name;
    private String biography;
    private String place_of_birth;
    private String profile_path;
    private String imdb_id;
    private String homepage;
}

File: app/src/main/java/com/aero51/moviedatabase/repository/model/tmdb/credits/Actor.kt
package com.aero51.moviedatabase.repository.model.tmdb.credits

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "actor")
class Actor (
    var birthday: String?,
    @PrimaryKey(autoGenerate = false)
    var id: Int,
    var name: String,
    var biography: String,
    var place_of_birth: String?,
    var profile_path: String?,
    var imdb_id: String?,
    var homepage: String?
){}